go/vt/sqlparser/sql.y

```diff
@@ -5694,7 +5711,16 @@ charset:
   {
     $$ = string($1)
   }
-
+| ASCII
+  {
+    // ASCII: Shorthand for CHARACTER SET latin1.
+    $$ = "ascii"
+  }
+| UNICODE
+  {
+    // UNICODE: Shorthand for CHARACTER SET ucs2.
+    $$ = "ucs2"
+  }
```

Comment:
These need to be added since a regular charset can also specify these but they'd otherwise be treated as tokens and not strings.